demos/colosseum/web/static/style.css

```
@@ -595,6 +595,157 @@ main {
+  background: #fff;
+}
+.preview-text {
+  margin: 0;
+  width: 100%;
+  overflow: auto;
+  font-family: var(--mono);
+  font-size: 0.85rem;
+  line-height: 1.6;
+  color: var(--fg);
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)